### PR TITLE
Changelog and manifest

### PR DESCRIPTION
**Pull request by sarenameas**
Add a template CHANGELOG.md and a manifest.yml in preparation for the next release.

### CODE REVIEW

**Review by leegeth — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Change Log for coreHTTP Client Library
+
+## v1.0.0 (MONTH YEAR)
+
+This is the first release of the coreHTTP client library in this repository.
+
+The HTTP client library is a client-side implementation that implements a subset
```
Comment (dan4thewin) [inline CHANGELOG.md:7]: An "implementation that implements"?  Maybe reword this?

Comment (sarenameas) [reply CHANGELOG.md:7]: lol sure.

**Review by dan4thewin — APPROVED** [review]

**Review by leegeth — APPROVED** [review]
